bin/heudiconv

```diff
@@ -238,14 +231,25 @@ def embed_nifti(dcmfiles, niftifile, infofile, force=False):
     #orig_nii.update_header()
     #orig_nii.to_filename(niftifile)
     meta = ds.NiftiWrapper(new_nii).meta_ext.to_json()
-    with open(infofile, 'wt') as fp:
+    with open(infofile, 'at') as fp:
```

Comment:
@satra essentially I add the info to the scaninfo file here, if bids option is called this is added after dcm2niix's, if not this is this only info in the file